src/store/index.js

```diff
@@ -17,7 +18,8 @@ export function init (store = {}, ...middleware) {
     ]);
 
     registry.register({
-        composes: composeReducer,
+        composes,
+        pendingCompose,
```

Comment:
I am not sure how well it will play with the clouddot ReducerRegistry but I believe it would be good to combine the reducers into one reducer using [combineReducers](https://redux.js.org/api/combinereducers). This can be done in an index.js file in the reducers directory like how we combine and export our actions in their index.js

Reply:
Right so the [ReducerRegistry](https://github.com/RedHatInsights/insights-frontend-components/blob/master/src/Utilities/ReducerRegistry.js) already combines any reducers we register.

I think that's also what allows us to just pass a `store` object to init() which follows the general store structure.